app/surveys/mci_0213.py

```diff
@@ -0,0 +1,20 @@
+import copy
+
+from app.surveys.mci_0215 import mci_0215
+
+eq_id = "1"
+form_type = "0213"
+
```

Comment:
Was done due to question numbering and position, if we created 0213 then added a question to get to 0215, getting the number and position in the json is far more complicated